h/views/activity.py

```diff
@@ -40,12 +40,6 @@ def search(request):
     # Fetch results
     result = query.execute(request, q, page_size=page_size)
 
-    for user in result.aggregations.get('users', []):
-        user['username'] = util.user.split_user(user['user'])['username']
-        user['userid'] = user['user']
-        user['faceted_by'] = _faceted_by_user(request, user['username'], q)
-        del user['user']
-
```

Comment:
Ah, you replaced this with a dict comprehension below, and also removed the need for `split_user()`. Nice refactoring!